packages/daimo-api/src/landline/connector.ts

```diff
@@ -119,3 +119,41 @@ export async function landlineDeposit(
     return { status: "error" };
   }
 }
+
+// TODO: remove all duplicate types across repos
+export type FastFinishRejectReason = "tx-limit" | "monthly-limit";
+export type ShouldFastFinishResponse = {
+  shouldFastFinish: boolean;
+  reason: FastFinishRejectReason | null;
+};
+
+/**
+ * Performs any necessary validations before initiating a Landline deposit.
+ * @param daimoAddress destination address of the deposit (Daimo account)
+ * @param amount amount in the units of the destination token (USDC)
+ * @returns
+ */
+export async function validateLandlineDeposit(
+  daimoAddress: Address,
+  amount: string
+) {
+  console.log("[LANDLINE] validating deposit", {
+    daimoAddress,
+    amount,
+  });
+  try {
+    const validateResponse: ShouldFastFinishResponse =
+      // @ts-ignore
+      await landlineTrpc.shouldFastDeposit.query({
+        daimoAddress,
+        amount,
+      });
+    console.log(
+      `[LANDLINE] validated deposit for ${daimoAddress}, amount: ${amount}`,
+      validateResponse
```

Comment:
i think this might just print `[object Object]`

consider something like:

```ts
export async function validateLandlineDeposit(args: {
  daimoAddress: Address,
  amount: string
}) {
  console.log(`[LANDLINE] validating deposit ${debugJson(args)}`);
   // ..
   console.log(`[LANDLINE] validateLandlineDeposit ${debugJson({ args, response })}`);
```